### PR TITLE
Specflow actions windows app driver

### PR DESCRIPTION
**Pull request by JordanWestern**

### CODE REVIEW

File: Plugins/SpecFlow.Actions.Configuration/SpecFlow.Actions.Configuration/SpecFlow.Actions.Configuration.csproj [hunk 1]
```diff
@@ -14,4 +14,8 @@
     <PackageReference Include="System.Text.Json" Version="5.0.2" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="Nerdbank.GitVersioning" Version="3.4.240" />
```
Comment (SabotageAndi) [inline Plugins/SpecFlow.Actions.Configuration/SpecFlow.Actions.Configuration/SpecFlow.Actions.Configuration.csproj:18]: This package is added via `Directory.Build.props` (https://github.com/SpecFlowOSS/SpecFlow.Actions/blob/main/Plugins/Directory.Build.props#L13). Please update there the version number

File: Plugins/SpecFlow.Actions.WindowsAppDriver/SpecFlow.Actions.WindowsAppDriver/SpecFlow.Actions.WindowsAppDriver.csproj [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <PackageReference Include="Appium.WebDriver" Version="4.3.1" />
+    <PackageReference Include="SpecFlow" Version="3.9.22" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\SpecFlow.Actions.Configuration\SpecFlow.Actions.Configuration\SpecFlow.Actions.Configuration.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Update="Nerdbank.GitVersioning" Version="3.4.240" />
```
Comment (SabotageAndi) [inline Plugins/SpecFlow.Actions.WindowsAppDriver/SpecFlow.Actions.WindowsAppDriver/SpecFlow.Actions.WindowsAppDriver.csproj:20]: some here

File: Plugins/SpecFlow.Actions.WindowsAppDriver/Example/Specflow.Actions.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "windowsAppDriver": {
+    "capabilities": {
+      "app": "../SpecFlowCalculator/bin/Debug/net5.0-windows/SpecFlowCalculator.exe"
+    },
+    "WindowsAppDriverPath": "{WINDOWS_APP_DRIVER_EXECUTABLE_PATH}"
```
Comment (SabotageAndi) [inline Plugins/SpecFlow.Actions.WindowsAppDriver/Example/Specflow.Actions.json:6]: Should this work? Do we have an environment variable expander for the configuration values? I didn't see any in the code.

**Review by SabotageAndi — CHANGES_REQUESTED** [review]

File: Plugins/SpecFlow.Actions.WindowsAppDriver/Example/Specflow.Actions.json (hunk 3 above)
Comment (JordanWestern) [reply Plugins/SpecFlow.Actions.WindowsAppDriver/Example/Specflow.Actions.json:6]: This is just a placeholder text has nothing to do with environment variables, its just to tell the user what is expected

File: Plugins/SpecFlow.Actions.Configuration/SpecFlow.Actions.Configuration/SpecFlow.Actions.Configuration.csproj (hunk 1 above)
Comment (JordanWestern) [reply Plugins/SpecFlow.Actions.Configuration/SpecFlow.Actions.Configuration/SpecFlow.Actions.Configuration.csproj:18]: Did that by mistake, removed

File: Plugins/SpecFlow.Actions.WindowsAppDriver/SpecFlow.Actions.WindowsAppDriver/SpecFlow.Actions.WindowsAppDriver.csproj (hunk 2 above)
Comment (JordanWestern) [reply Plugins/SpecFlow.Actions.WindowsAppDriver/SpecFlow.Actions.WindowsAppDriver/SpecFlow.Actions.WindowsAppDriver.csproj]: Did that by mistake, removed

**Comment by JordanWestern** [comment]
@SabotageAndi  the build 'SpecFlow-Actions-WinAppDriver' was me figuring stuff out in AzDo, can be ignored as it has been removed

**Review by SabotageAndi — APPROVED** [review]

**Pull request merged**